.buildkite/integration.pipeline.yml

```diff
@@ -126,13 +135,22 @@ steps:
           PACKAGES: "docker"
           PLATFORMS: "linux/amd64"
           FIPS: "true"
-        command: ".buildkite/scripts/steps/integration-package.sh"
+          CUSTOM_IMAGE_TAG: "git-${BUILDKITE_COMMIT:0:12}"
+          CI_ELASTIC_AGENT_DOCKER_IMAGE: "docker.elastic.co/beats-ci/elastic-agent-cloud-fips"
+        command: |
+          source .buildkite/scripts/common.sh
+          .buildkite/scripts/steps/integration-package.sh
+          echo "~~~ Pushing cloud image"
+          mage cloud:push
```

Comment:
This duplicates the command from L102, I think it can be better to use a common script